documentation/book/con-kafka-listeners.adoc

```diff
@@ -55,6 +55,33 @@ endif::Kubernetes[]
 
 When exposing Kafka using {OpenShiftName} `Routes`, TLS encryption is always used.
 
+By default, the route hosts will be assigned by {OpenShiftName}.
+You can override the auto assigned route hosts by specifying the requested host in the `overrides` property.
+{ProductName} will not perform any validation that the requested hosts are available.
+You have to make sure that the hosts are free and can be used.
+
+.An example where the external listener is configured with overrides for {OpenShiftName} route hosts
+[source,yaml,subs="attributes+"]
+----
+# ...
+listeners:
+  external:
+    type: routes
+    authentication:
+      type: tls
+    overrides:
+      bootstrap:
+        host: bootstrap.myrouter.com
+      brokers:
+      - broker: 0
+        host: broker-0.myrouter.com
+      - broker: 1
+        host: broker-1.myrouter.com
+      - broker: 2
+        host: broker-2.myrouter.com
+# ...
+----
+
 For more information on using `Routes` to access Kafka, see xref:proc-accessing-kafka-using-routes-{context}[].
```

Comment:
As already done for the following section about nodeports, should be say the same about `bootstrap`.
I am referring to the paragraph : 

> Additionally, you can specify the name of the bootstrap service.
This name will be added to the broker certificates and can be used for TLS hostname verification.

Reply:
That is not in the NodePort section. There is a section at the end called `Customising advertised addresses on external listeners` which applies to all external listeners.

Reply:
I hope the clarification sentence which I'm adding as part of the third comment will make it more clear.